python/create_proxy_jwt.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2022 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+
+import argparse
+import sys
+import jwt
+import datetime
+from datetime import datetime
+from datetime import timezone
+from datetime import timedelta
```

Comment:
Could combine all the "from datetime import..." into one line, but it may also be personal preference to leave them in separate lines.